source/MaterialXView/Viewer.cpp

```diff
@@ -70,9 +70,10 @@ void writeTextFile(const std::string& text, const std::string& filePath)
     file.close();
 }
 
-mx::DocumentPtr loadLibraries(const mx::StringVec& libraryFolders, const mx::FileSearchPath& searchPath)
+std::pair<mx::DocumentPtr, mx::StringVec> loadLibraries(const mx::StringVec& libraryFolders, const mx::FileSearchPath& searchPath)
```

Comment:
Since the set of referenced source URI's can be determined from the Document itself, it might be better to add a core helper method along the lines of Document::getReferencedSourceUris, rather than extending this return value to a pair.  That helper method would potentially be useful in other situations as well, and it would make for a clearer interface to this function.